class-6-js-dom/khadija-bilal-jsdom-andarray/script.js

```diff
@@ -0,0 +1,64 @@
+/*
+
+ https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction
+ "Core Interfaces in the DOM"
+ 
+ The following is a brief list of common APIs in web and XML page scripting using the DOM:
+
+document.getElementById(id)
+document.getElementsByTagName(name)
+document.createElement(name)
+document.querySelector(selector)
+parentNode.appendChild(node)
+element.innerHTML
+element.style.left
+element.setAttribute()
+element.getAttribute()
+element.addEventListener()
+window.content
+window.onload
+window.scrollTo()
+*/
+
+// Challange 1:
+const header2 = document.getElementById("header");
+console.log(header2.textContent);
+header2.style.textAlign = "center";
+
+// Challang 2:
+const span = document.getElementById("spanCounter");
+const input = document.getElementById("inputId");
+const incButton = document.getElementById('increaseButton');
+
+function increaseCounter() {
+  if (input.value.length == 0){
```

Comment:
this is working, but this isn't a best practice, if the inout is number, .length wouldn't work, I would suggest trying a more secure condition